packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -1389,10 +1398,11 @@ abstract class BaseVolumeViewport extends Viewport {
       return;
     }
 
-    const { uid: defaultActorUID } = defaultActor;
-    volumeId = volumeId ?? defaultActorUID;
+    volumeId = volumeId ?? this.getVolumeId();
```

Comment:
I think this should probably be || - we really don't want to allow an empty string or null to be the Id, just seems like a bad idea.